cli/tests/integration/lsp_tests.rs

```diff
@@ -1871,7 +1871,7 @@ fn lsp_hover_dependency() {
     Some(json!({
       "contents": {
         "kind": "markdown",
-        "value": "**Resolved Dependency**\n\n**Code**: http&#8203;://127.0.0.1:4545/xTypeScriptTypes.js\n"
+        "value": "**Resolved Dependency**\n\n**Code**: http&#8203;://127.0.0.1:4545/xTypeScriptTypes.js\n\n**Types**: http&#8203;://127.0.0.1:4545/xTypeScriptTypes.d.ts\n"
```

Comment:
I'm not really sure how my changes fixed this, but I'll take it.

Reply:
I think the loader you passed when creating a graph can discover type dependencies, but the default one can't